roles/matrix-synapse/templates/ext/appservice-irc/systemd/matrix-appservice-irc.service.j2

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=Matrix Appservice IRC server
+After=docker.service
+Requires=docker.service
+Requires=matrix-synapse.service
+After=matrix-synapse.service
+
+[Service]
+Type=simple
+ExecStartPre=-/usr/bin/docker kill matrix-appservice-irc
+ExecStartPre=-/usr/bin/docker rm matrix-appservice-irc
+ExecStart=/usr/bin/docker run --rm --name matrix-appservice-irc \
+			--log-driver=none \
+			-e "UID={{ matrix_user_uid }}" -e "GID={{ matrix_user_gid }}" \
```

Comment:
We could probably do with the rootless stuff and dropping capabilities in another PR later.

Reply:
Sounds good.